test/cmp_hdr_test.c

```diff
@@ -323,19 +332,24 @@ execute_HDR_generalInfo_push1_items_test(CMP_HDR_TEST_FIXTURE *fixture)
 
     if (!TEST_int_eq(ossl_cmp_hdr_generalInfo_push1_items(fixture->hdr, itavs),
                      1))
-        return 0;
+        goto err;
```

Comment:
You need to `asn1int = NULL; val = NULL;` above otherwise you'll doublefree in the err cases.

Reply:
Good catch.  Will fix.